dds_web/utils.py

```diff
@@ -389,3 +389,24 @@ def create_one_time_password_email(user, hotp_value):
     )
 
     return msg
+
+
+def bucket_is_valid(bucket_name):
+    """Verify that the bucket name is valid."""
+    valid = False
+    message = ""
+    if not (3 <= len(bucket_name) <= 63):
+        message = f"The bucket name has the incorrect length {len(bucket_name)}"
+    elif re.findall(r"[^a-zA-Z0-9.-]", bucket_name):
+        message = "The bucket name contains invalid characters."
+    elif bucket_name[0] in [".", "-"]:
```

Comment:
if anything else than a letter or a number is not allowed, then this check should not be enough, maybe this:

```suggestion
    elif bucket_name[0].isalnum():
```

if anything else than a letter or a number is not allowed